sklearn/multiclass.py

```diff
@@ -437,6 +443,11 @@ def multilabel_(self):
     def n_classes_(self):
         return len(self.classes_)
 
+    # TODO: Remove in 0.26 when the coef_ attribute is deprecated
+    # mypy error: Decorated property not supported
+    @deprecated("Attribute coef_ was deprecated in "  # type: ignore
+                "version 0.24 and will be removed in 0.26. "
+                "Use the importance_getter parameter instead.")
```

Comment:
We should specify "If you observe this warning while using RFE or SelectFromModel, use..."

and we should also indicate that in the docs warning